### ISSUE TITLE
Helpful changes to MiniFlow Lesson - Gradient Descent Part 2

### ISSUE THREAD

**Issue by sahiljuneja** [udacity/sdc-issue-reports#163]
Could the image actually have labels for H,F, and G on them? I believe that would be helpful to an extent. 

Secondly, this applies to the entire course perhaps, could there be a slight difference in "Equation Labels"?

Like the equations which have something like "Equation 6" underneath them, could be in bold?

It's easier to locate them if the text refers to an equation at some point because not all mathematical expressions in the course have an equation number associated with them.

**Comment by domluna** [comment]
@sahiljuneja. That section and the backprop section have been redone from the ground up so this no longer applies.
